### DISCUSSION TITLE
Add existing or create new item in one step

### DISCUSSION THREAD

**Discussion by lasseklein** [directus/directus#18650] [Feature Requests]
### Summary

Combine the actions "Create new" and "Add existing" into a single button that contains both functions to simplify entering items when you don't know if they exist or need to be entered.

### Basic Example

Airtable solves this by combining the action into one: You enter a list of existing items, and there's a button to add a new one. If you start writing, the list is filtered, and when there are no matches left, the remaining option is to create a new item with the name you have entered. Pretty sweet.

<img width="594" alt="Screenshot 2023-05-18 at 20 07 49" src="https://github.com/directus/directus/assets/685735/217fefa6-13a6-4e16-a343-728e0a1993b2">
<img width="597" alt="Screenshot 2023-05-18 at 20 07 37" src="https://github.com/directus/directus/assets/685735/510c1399-0063-45e9-9eae-5bea9b9facc5">

### Motivation

Today, adding a reference object involves choosing between "Create new" or "Add existing". If there are many items, it can be challenging to know if you need to select an existing item or add a new one, so there is a bit back and forth to first check, then exit and add a new one.

<img width="409" alt="Screenshot 2023-05-18 at 20 15 47" src="https://github.com/directus/directus/assets/685735/9b664423-6a9e-431c-b1dc-743b1b28dd09">
<img width="791" alt="Screenshot 2023-05-18 at 20 03 55" src="https://github.com/directus/directus/assets/685735/ecff71c9-56c4-4777-abe7-6879e97ae1ce">

### Detailed Design

Replace the two buttons "Create new" and "Add existing" with a single "+" button. 

Pressing plus opens a view for selecting existing items as today, but with a new option of adding a new item, and also a field for filtering the existing items.

It could still be possible to configure the field to accept only adding or selecting existing items

### Should Have:
- Single button for adding a reference to the list
- Functionality to select existing or add new item in the same view

### Drawbacks

May need more people desiring the same simplification

### Alternatives

Content editors are often concerned with making their workflow as effective as possible, especially for repeating tasks like this one.

### Adoption Strategy

Should be intuitive to get used to for existing editors.

### Unresolved Questions

May need more insight from the community.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by emahuni** [comment]
I think this is a simple tweak that needs to be implemented considering the user interface implications on usage. Please consider this.
